<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyCarDishShowBar</class>
 <widget class="QWidget" name="BuyCarDishShowBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="topWin" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#topWin
{    
	background-color: rgb(179,179,164,20%); /* 设置背景颜色 */
    border: 1px solid #c0c0c0; /* 设置边框 */
    border-radius: 5px; /* 设置圆角 */
    padding: 5px 10px; /* 设置内边距 */
}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="DishName_lab">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Dishmoney_lab">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DishBuyNum_lab">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="allMoney_lab">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="DishShow_lab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="select_chkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选中</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
